### PR TITLE
Merge Windows Debug-Build into Release-Build

### PR DESCRIPTION
**Pull request by arogge**
As the difference between debug and release for Windows is the shipping of the sourcecode only, this PR merges both packages into one.
The Sourcecode is now an optional component that can be selected at install-time.

To reconfigure the builds, this PR also adds a .matrix.yml that will tell Jenkins to only do the release-builds in the future.

#### Please check

- [x] Short description and the purpose of this PR is present _above this paragraph_
- [x] Your name is present in the AUTHORS file (optional)

If you have any questions or problems, please give a comment in the PR.

### Helpful documentation and best practices

- [Git Workflow](https://docs.bareos.org/DeveloperGuide/gitworkflow.html)
- [Automatic Sourcecode Formatting](https://docs.bareos.org/DeveloperGuide/generaldevel.html#automatic-sourcecode-formatting)
- [Check your commit messages](https://docs.bareos.org/DeveloperGuide/gitworkflow.html#commits)

##### General

- [x] PR name is meaningful
- [x] Purpose of the PR is understood
- [x] Separate commit for this PR in the CHANGELOG.md, PR number referenced is same
- [x] Commit descriptions are understandable and well formatted

##### Source code quality

- [x] Source code changes are understandable
- [x] Variable and function names are meaningful
- [x] Code comments are correct (logically and spelling)
- [x] Required documentation changes are present and part of the PR
- [x] `bareos-check-sources --since-merge` does not report any problems
- [x] `git status` should not report modifications in the source tree after building and testing

### CODE REVIEW

File: .matrix.yml [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+    "Leap_15.2":
+      TYPE: rpm
+      IMAGE: opensuse-leap152
+      ARCH:
+      - x86_64
+      PROJECTPACKAGES:
+       x86_64:
+          - bareos
+          - python-bareos
+    "4.4":
```
Comment (pstorz) [inline .matrix.yml:41]: I think here is something missing, probably 
**Univention:**

**Review by pstorz — CHANGES_REQUESTED** [review]
The matrix.yml file is not correct.

**Review by pstorz — CHANGES_REQUESTED** [review]
I think it makes sense to add a line to the CHANGELOG that the two windows builds were merged and only one Installer is now available for windows 32 and for windows 64.

**Comment by arogge** [comment]
I triple-checked that all distros are built, all files are still produced as desired (i.e. no webui tarballs) and added the CHANGELOG record you requested.

**Review by pstorz — APPROVED** [review]
Good work, looks good to me. You might want to squash the last two commits.

**Pull request merged**